cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma

```diff
@@ -366,3 +372,37 @@ static stock CmpVersions(const a[], const b[]) {
 
   return countA - countB
 }
+
+stock log_to_file_ex(const filepath[], message[]) {
+  new iFile, bool:bFirstTime = true, szDate[32]
```

Comment:
Please follow the codestyle of the file.
 - Use the simpler [camelCase](https://en.wikipedia.org/wiki/Camel_case), instead of [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)